src/components/ContentPreview/ContentPreview.js

```diff
@@ -604,7 +612,39 @@ class ContentPreview extends PureComponent<Props, State> {
      */
     fetchFileSuccessCallback = (file: BoxItem): void => {
         this.fetchFileEndTime = performance.now();
-        this.setState({ file });
+        this.retryCount = 0;
+        this.setState({ file, isFileError: false });
+    };
+
+    /**
+     * File fetch error callback
+     *
+     * @return {void}
+     */
+    /* eslint-disable no-unused-vars */
+    fetchFileErrorCallback = (e: Error): void => {
```

Comment:
@DanDeMicco do we need to do anything to the file fetch end time if the files call fails after retries?